Shuttle-back/src/main/java/com/shuttle/driver/DriverService.java

```diff
@@ -31,6 +41,50 @@ public Driver setAvailable(Driver driver, boolean available) {
 		
 		return driver;
 	}
+
+	@Override
+	public List<LocationDTO> getActiveDriversLocations() {
+		List<Driver> activeDrivers = driverRepository.findByAvailableTrue();
+		List<Vehicle> driversVehicles = vehicleRepository.findByDriverIn(activeDrivers);
+		return driversVehicles.stream().map(vehicle -> LocationDTO.from(vehicle.getCurrentLocation())).toList();
+	}
+
+	@Override
+	public boolean changeCurrentLocation(long driverId, LocationDTO location) {
+		Optional<Driver> driver = this.driverRepository.findById(driverId);
+		if(driver.isPresent()) {
+			Vehicle vehicle = vehicleRepository.findByDriver(driver.get());
+			vehicle.setCurrentLocation(location.to());
+			this.vehicleRepository.save(vehicle);
```

Comment:
Again, these three lines should be `vehicleService.setCurrentLocation(Driver driver, Location newLocation)`, so create the method in `VehicleService`.